pom.xml

```diff
@@ -84,7 +84,7 @@
         <infinispan.version>13.0.10.Final</infinispan.version>
         <infinispan.protostream.processor.version>4.4.1.Final</infinispan.protostream.processor.version>
         <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
-        <jackson.version>2.13.4</jackson.version>
+        <jackson.version>2.13.4.20221013</jackson.version>
```

Comment:
Looks like we need here `2.13.4.2` as per https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.13.

Did you use the script to upgrade Quarkus? If so, we should probably fix it to not get the version from the Jackson BOM.

Reply:
Now that you mentioned, I will definitely use the script. I have one small question. The reason why I updated jackson to `2.13.4.20221013` was to keep those versions in sync with Quarkus. See:  https://github.com/quarkusio/quarkus/blob/2.13.3.Final/bom/application/pom.xml#L86.

Do you think it is ok to update to `2.13.4.2`?